web/src/components/Settings/SystemSection.tsx

```diff
@@ -60,6 +60,22 @@ const SystemSection = () => {
     });
   };
 
+  const handleVacuumBtnClick = async () => {
+    try {
+      api.vacuumDatabase().then(({ data }) => {
```

Comment:
We should call the system status API again instead of returning filesize in VACUUM API.